main.go

```diff
@@ -161,8 +161,16 @@ func Run(client *cmd.Client, args ...string) {
 		{
 			Name:    "withdraw",
 			Aliases: []string{"w"},
-			Usage:   "Withdraw LINK to an authorized address",
-			Action:  client.Withdraw,
+			Usage:   "Withdraw, to an authorized Ethereum <address>, <amount> units of LINK. Withdraws from the configured oracle contract by default, or from contract optionally specified by a third command-line argument --from-oracle-contract-address=<contract address>. Address inputs must be in EIP55-compliant capitalization.",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "from-oracle-contract-address",
+					Value: "",
```

Comment:
✂️ 
Remove `Value` if it's going to be empty.

Reply:
2d4dabc